### ISSUE TITLE
Please add a license file and screen shots

### ISSUE THREAD

**Issue by unixorn** [JtMotoX/zsh-jt-themes#1]
I'd like to add this to [awesome-zsh-plugins](https://github.com/unixorn/awesome-zsh-plugins), but I only add themes with screen shots in the readme so that users can see what they look like without having to install them.

Also, please add a license file, some people won't use any code that doesn't have an open source license. If you don't have one in mind, https://choosealicense.com is a good resource to help you select an appropriate one.

Finally, please consider breaking the individual themes into separate repositories - it makes it easier for people using ZSH frameworks to load the theme.

Thanks!

**Comment by JtMotoX** [comment]
Hello @unixorn.  I have visited your repo quite a bit when switching from bash to zsh.  I really appreciate all your research and work.

I have added a [README](https://github.com/JtMotoX/zsh-jt-themes/blob/main/README.md) that explains my intentions for this repo.  Essentially, I do not think anybody will find these useful and do not think these belong on your list of themes and plugins.  Again, I really appreciate your work, and thanks for reaching out.  Keep it up! 👍
